src/service/resources-impl.js

```diff
@@ -784,7 +794,10 @@ export class Resources {
             minTop = minTop == -1 ? box.top : Math.min(minTop, box.top);
           }
           request.resource./*OK*/changeSize(
-              request.newHeight, request.newWidth, request.callback);
+              request.newHeight, request.newWidth);
+          if (request.callback) {
+            request.callback(/* hasSizeChanged */true);
+          }
           request.resource.overflowCallback(/* overflown */ false,
               request.newHeight, request.newWidth);
```

Comment:
https://github.com/ampproject/amphtml/blob/master/src/custom-element.js#L1296
https://github.com/ampproject/amphtml/blob/master/src/service/resource.js#L280
If I remove width and height, what I do with the these?